lib/KTable/KTableGridItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <td
+    :class="$computedClass(coreOutlineFocus)"
+    :style="{ textAlign: textAlign, minWidth: minWidth, width: width }"
+    tabindex="0"
+    data-focus="true"
+    role="gridcell"
+    @keydown="onKeydown"
+  >
+    <slot :content="content">
+      {{ content }}
+    </slot>
+  </td>
+
+</template>
+
+
+<script>
+
+  import { DATA_TYPE_NUMERIC } from './useSorting';
+
+  export default {
+    name: 'KTableGridItem',
+    props: {
+      content: {
+        type: [String, Number],
+        required: true,
+      },
+      dataType: {
+        type: String,
+        default: 'string',
+      },
+      rowIndex: {
+        type: Number,
+        required: true,
+      },
+      colIndex: {
+        type: Number,
+        required: true,
+      },
+      minWidth: {
+        type: String,
+        default: 'auto',
+      },
+      width: {
+        type: String,
+        default: 'auto',
+      },
+    },
+    computed: {
+      coreOutlineFocus() {
+        return {
+          ':focus': {
+            ...this.$coreOutline,
+            outlineOffset: '-2px',
+          },
+        };
+      },
+      textAlign() {
+        const alignLtr = this.dataType === DATA_TYPE_NUMERIC ? 'right' : 'left';
```

Comment:
Is there a particular reason numeric data types are right aligned compared to other data types? I'm curious about this because there seems to be some misalignment compared to the centered headers when creating classes in the Kolibri pr: 
![classListAlignment](https://github.com/user-attachments/assets/6db45f2a-0ef6-4fa5-984d-5b87cec202d5)

Reply:
This was inspired from the MS Excel default alignment for numbers. In MS Excel, text and other datatypes are aligned to the left while numbers are aligned to the right. This ideally helps with :
- Enhanced readability since numbers are generally read from right-to-left. So, this makes numbers of different lengths to be visualised/compared easily.  
- Better visibility as can be distinguished from other datatypes.
As an example, in the below image, data presented in the right-aligned format seems more readable than the data aligned in left format.
![image](https://github.com/user-attachments/assets/5e698cb5-2435-4db5-b80a-cfcdc4445479)

Reply:
Okay!

Reply:
@LianaHarris360 and @MisRob thank you for pointing this out. I noticed this as well and wondered if it was intentional. I do think that table headers should be aligned with the alignment of the data, and never be center-aligned since it makes it a more difficult to tell which header is associated with which data.

Reply:
Thank you @jtamiace. That will help, yes. And should we always align to the left (right in RTL) or would you prefer to keep the numeric type on the right?

Reply:
Let's keep numeric type on the right

Reply:
:+1: Thanks, all clear :) @BabyElias I know we originally discussed doing this in the follow-up, but now wondering if it'd make sense to have it here since it seems to be a simple change with a pretty high impact on the user experience and we have several days before the new release? It also depends on other work you have in progress. So no pressure, rather if it doesn't cause much trouble.

Reply:
I think doing this change in the current PR itself would be better. Working on it!

Reply:
Lovely, thank you